packaging_automation/common_tool_methods.py

```diff
@@ -398,7 +398,7 @@ def delete_all_gpg_keys_by_name(name: str):
     delete_public_gpg_key_by_name(name)
 
 
-def get_secret_key_by_fingerprint_without_password(fingerprint: str) -> str:
+def get_private_key_by_fingerprint_without_password(fingerprint: str) -> str:
```

Comment:
```suggestion
def get_private_key_by_fingerprint_without_passphrase(fingerprint: str) -> str:
```

Passphrase and password are slightly different things. Let's keep the terminology correct here.

Please correct similar occurrences as well.

Reply:
I already changed the lines can you see it
https://github.com/citusdata/tools/blob/66a5602b7130283a3605c4df6a235d63601b2338/packaging_automation/common_tool_methods.py#L407-L428

Reply:
There was a problem in the function name 'get_private_key_key_by_fingerprint_with_passphrase' changed it into get_private_key__by_fingerprint_with_passphrase